packages/teleport/src/Apps/AddApp/Automatically/Automatically.tsx

```diff
@@ -25,8 +25,8 @@ import {
   ButtonPrimary,
 } from 'design';
 import Validation, { Validator } from 'shared/components/Validation';
-import FieldInput from 'shared/components/FieldInput';
 import { DialogContent, DialogFooter } from 'design/Dialog';
+import FieldInput from 'shared/components/FieldInput';
```

Comment:
i don't think we pay close attention to this, but when we do: https://gravitational.slack.com/archives/C02DQ1C2BMW/p1643999171617869